README.md

```diff
@@ -161,42 +151,47 @@ const useModal = () => {
         top: 50%;
         transform: translate(-50%,-50%);
         z-index: 1000;
-      `
-    }
-  })
+      `;
+    },
+  });
 
   return {
     Modal: Portal,
     openModal: openPortal,
     toggleModal: togglePortal,
     closeModal: closePortal,
-    isOpen
-  }
-}
+    isOpen,
+  };
+};
 
 const App = () => {
-  const { openModal, closeModal, isOpen, Modal } = useModal()
-  
-  return <>
-    <button onClick={e => openModal(e)}>Open Modal<button>
```

Comment:
Why not just change this line then? I'm not a fan of `;` or `"`

Reply:
Yeah, I am gonna re-push that change only. Sorry, bad format on save.